### PR TITLE
Add GraphQL api

### PR DESCRIPTION
**Pull request by frahugo**

### CODE REVIEW

**Comment by frahugo** [comment]
@nicholasjhenry I added a `person` query in the schema.  Have a look, and if OK with that, we could merge into master and deploy to Heroku (in case you'd like to show that to Thann tomorrow).

File: apps/loud_mouth_graphql/lib/loud_mouth_graphql/directory/schema.ex [hunk 1]
```diff
@@ -4,14 +4,15 @@ defmodule LoudMouthGraphQL.Directory.Schema do
...
+    @desc "List of people"
     field :people, list_of(:person) do
       resolve &PersonResolver.all/2
     end
   end
 
   mutation do
     @desc "Add a person"
-    field :person, type: :person do
+    field :add_person, type: :person do
```
Comment (nicholasjhenry) [inline apps/loud_mouth_graphql/lib/loud_mouth_graphql/directory/schema.ex]: 👍 that makes more sense!

File: apps/loud_mouth_graphql/mix.exs [hunk 2]
```diff
@@ -44,7 +44,9 @@ defmodule LoudMouthGraphQL.Mixfile do
       # Domain
       {:loud_mouth, in_umbrella: true},
       # Plug support for Absinthe, GraphQL for Elixir
-      {:absinthe_plug, "~> 1.2.0-alpha0"}
+      {:absinthe_plug, "~> 1.2.0-alpha0"},
+      # Automatically run your Elixir project's tests each time you save a file
+      {:mix_test_watch, "~> 0.2", only: :dev}
```
Comment (nicholasjhenry) [inline apps/loud_mouth_graphql/mix.exs]: Could this be added to the route dependencies? i.e. `loud_mouth_platform/mix.exs`?

File: apps/loud_mouth_graphql/lib/loud_mouth_graphql/directory/schema.ex [hunk 3]
```diff
@@ -8,6 +8,13 @@ defmodule LoudMouthGraphQL.Directory.Schema do
     field :people, list_of(:person) do
       resolve &PersonResolver.all/2
     end
+
+    @desc "Get a person by email"
```
Comment (nicholasjhenry) [inline apps/loud_mouth_graphql/lib/loud_mouth_graphql/directory/schema.ex]: This is good for now, but it seems we would want to search on multiple facets i.e. `family_name`

**Comment by nicholasjhenry** [comment]
Excellent, I'll merge this in and deploy to Heroku! 👍

File: apps/loud_mouth_graphql/mix.exs (hunk 2 above)
Comment (frahugo) [inline apps/loud_mouth_graphql/mix.exs]: We have the dependency in the umbrella mix.exs.  I don't know how we could chain it with the inner mix.exs.

**Comment by nicholasjhenry** [comment]
@frahugo yeah, that's pretty cool 

screenshot 2016-11-03 17 59 36

**Comment by nicholasjhenry** [comment]
@frahugo Ah, right -- deps are not inherited from the umbrella, wondering
if we could have a `loud_mouth_test_support` app that includes the
dependencies we would want across the entire platform.

**Pull request merged**